.github/workflows/versioning.yml

```diff
@@ -0,0 +1,14 @@
+name: Keep major version tags up-to-date
+
+on:
+  release:
+    types: [published, edited]
+
+jobs:
+  actions-tagger:
+    runs-on: windows-latest
```

Comment:
The Windows runners are significantly slower than the Linux runners in my experience. Can we change that to `ubuntu-latest`? 
It [should work](https://github.com/bcanseco/github-contribution-graph-action/blob/31a07d74b3430073cd5dd4fcc7238464e9f7d0d9/.github/workflows/release.yml).

Reply:
You are right, it does: https://github.com/bcanseco/github-contribution-graph-action/actions/runs/3981711788/workflow

Thanks for linking the example. I was already wondering why windows was particularly used in the example...